scripts/test-runner.js

```diff
@@ -328,22 +328,8 @@ function getErrorHandlerWithWarningCapture(
       errorCodeSet.add(diagnostic.errorCode);
     }
     try {
-      switch (diagnostic.severity) {
-        case "Information":
-          if (verbose && !suppressDiagnostics) console.log(`Info: ${msg}`);
-          return "Recover";
-        case "Warning":
-          if (verbose && !suppressDiagnostics) console.warn(`Warn: ${msg}`);
-          return "Recover";
-        case "RecoverableError":
-          if (verbose && !suppressDiagnostics) console.error(`Error: ${msg}`);
-          return "Recover";
-        case "FatalError":
-          if (verbose && !suppressDiagnostics) console.error(`Fatal Error: ${msg}`);
-          return "Fail";
-        default:
-          invariant(false, "Unexpected error type");
-      }
+      if (verbose && !suppressDiagnostics) console.log(`${diagnostic.severity}: ${msg}`);
+      return "Recover";
```

Comment:
If the severity is a FatalError there is no possibility of recovery and returning "Recover" should be ignored. Returning it anyway may well be a valid thing for a test runner to do, as long as there are never any tests that depend on actual recovery. In other words, such test should fail not succeed.

Reply:
The tests won't "pass" when run though. CI would say that all of these new tests failed (which was the issue I kept running into before).